content/en/blog/2024/scaling-opentelemetry-collectors.md

```diff
@@ -175,12 +153,11 @@ Deploy the OpenTelemetry Collector across your hosts by executing:
 ansible-playbook deploy-opentelemetry.yml
 ```
 
+## Visualizing metrics in Grafana
 
-## Visualizing metrics and logs in Grafana
-
-With data successfully ingested into Grafana Cloud, you can create custom dashboards to visualize the metrics, logs, and traces received from your OpenTelemetry Collector. Utilize Grafana's powerful query builder and visualization tools to derive insights from your data effectively.
+With data successfully ingested into Prometheus, you can use Grafana to create custom dashboards to visualize the metrics received from your OpenTelemetry Collector's. Utilize Grafana's powerful query builder and visualization tools to derive insights from your data effectively.
 
 - Consider creating dashboards that offer a comprehensive overview of your infrastructure's health and performance.
 - Utilize Grafana's alerting features to proactively manage and respond to issues identified through the OpenTelemetry data.
 
-This guide simplifies the deployment of the OpenTelemetry Collector across multiple Linux hosts using Ansible and illustrates how to visualize collected telemetry in Grafana Cloud. Tailor the Ansible roles, OpenTelemetry Collector configurations, and Grafana dashboards to suit your specific monitoring and observability requirements.
+This guide simplifies the deployment of the OpenTelemetry Collector across multiple Linux hosts using Ansible and illustrates how to visualize collected telemetry in Grafana. Tailor the OpenTelemetry Collector Ansible roles configurations, and Grafana dashboards to suit your specific monitoring and observability requirements.
```

Comment:
this still reads like a marketing blob, I have no issues with promoting OSS grafana here, but ideally this is practical, so how to setup grafana swiftly to see some of the data.

Reply:
yes, exactly.